### PR TITLE
Eliminate return only generic in gapi.client

### PR DESCRIPTION
**Pull request by nreid260**
This typing construct adds no information.

### CODE REVIEW

File: types/gapi.client/index.d.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ declare namespace gapi {
             //  */
             // callback?: () => any;
         // tslint:disable-next-line:no-unnecessary-generics
```
Comment (Maxim-Mazurok) [inline types/gapi.client/index.d.ts:52]: I think this comment shoule be removed now

**Review by Maxim-Mazurok — CHANGES_REQUESTED** [review]
Good point, I was hesitant to do this. That's why I disabled warning with comment.
Could you also do the same for `function newBatch<T>(): Batch<T>;` in this file, while we're at it?

File: types/gapi.client/index.d.ts (hunk 1 above)
Comment (nreid260) [reply types/gapi.client/index.d.ts]: Done

**Review by Maxim-Mazurok — APPROVED** [review]
Thanks!

**Comment by nreid260** [comment]
Ping

**Pull request merged**